Firebase/Core/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# v6.1.0 -- M52
+# v6.2.0 -- M53
+- [added] Added AppKit dependency on macOS. (#3459)
+- [changed] Moved core diagnostics log to app launch when core data collection is enabled. (#3437)
+
```

Comment:
Should we say here that CoreDiagnostics logging is now open sourced with a pointer to the new documentation?

Reply:
Yes, because it breaks compatibility if mixed with older versions of core components (like in RC1). I couldn't find the PR/issue link for it though.

Reply:
Added. I'm not familiar with all the cases where the CoreDiagnostics changes caused breakages, so I haven't specified them here.